.github/workflows/codeql.yml

```diff
@@ -0,0 +1,82 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
```

Comment:
Let's remove this programmatically generated comment.

Reply:
https://github.com/cloudposse/atmos/pull/452/commits/ad2121dd2fee9c735cae7439a37a84a5e303a714

addressed!